f2/src/forms/ContactInfoForm.js

```diff
@@ -123,22 +123,55 @@ export const ContactInfoForm = (props) => {
                           id="email"
                         />
                         {errors.phone && (
-                          <P
-                            color="#dc3545"
-                            fontSize="1.25rem"
-                            marginBottom="0.5rem"
-                          >
-                            {phone.errorMessage}
-                          </P>
+                          <Container>
```

Comment:
Why are we adding containers everywhere?

Reply:
Advice taken, I will remove extra Container tag